### ISSUE TITLE
[A11y Audit]: CPAP Supply Reordering from Health Portfolio

### ISSUE THREAD

**Issue by eli-oat** [department-of-veterans-affairs/va.gov-team#62924]
### Auditor(s)

@coforma-jamie and @SarahKay8 

### Automated

- [x] Run axe checks on every page or unique state (required)
- [ ] Run WAVE on every page if time allows. WAVE is more visual than axe or axe-coconut, but offers some excellent ways to identify nested headings and HTML5 landmark tags.

### Manual

- [x] Color contrast checks (recommend using whocanuse.com for beyond compliance reports)
- [x] Color blindness checks
- [x] Zoom layouts to 400% and inspect them for readability. If layouts break at 400%, I will start reducing them until they become stable, and log the zoom ratio when things started breaking.
- [x] Keyboard navigation

### Screen Reader

- [x] Firefox and NVDA
- [x] Edge and JAWS
- [ ] iOS Safari and VoiceOver (Mobile) `no device`
- [x] Safari and Voiceover (Desktop)
- [x] Chrome and Talkback

### Staging or Production Link

[Entry point](https://staging.va.gov/health-care/order-cpap-supplies/)

### Test Users

```
1. vets.gov.user+228@gmail.com
```

### Audit spreadsheet
[Audit spreadsheet](https://docs.google.com/spreadsheets/d/1-kWUnZN_dvHGLfiL1vhL5mANMsKlSXDmjhaQxTi07c4/edit?usp=sharing)

```[tasklist]
### CAIA Tasks
- [x] Add flows to audit spreadsheet
- [x] Jamie - ask SK for mobile testing
- [x] Complete audit
- [x] Share findings with team
- [ ] Answer questions from team
- [ ] If team is able to create another test user with CPAP ordering capability, test that flow
```

```[tasklist]
### Tied to
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/61624
```

**Comment by sara-amanda** [comment]
## CAIA Updates
### Intake Ticket Info. (Quick Ref.)

- **<Epic> CPAP Supply Reordering from Health Portfolio Ticket: [61624](https://app.zenhub.com/workspaces/sitewide-content-accessibility-and-ia-63a1d63232beba0011a7833f/issues/gh/department-of-veterans-affairs/va.gov-team/61624)**
- **Content, Accessibility, and IA work needed by:** August 2023
- **Contact:** Jasmine Yohannan (product manager)

> We are a team focused on adding CPAP machines to the hearing aid and batteries tool by the end of summer 2023. Following this we will be looking to add other features/supplies to this tool.

> My team needs support with: 1. How to update the H1s and any other content to accommodate adding CPAP machines to the hearing aid and batteries reorder tool. 2. This is after initial launch, but we need to decide if we need to break apart prosthetic socks with the growing universe of supplies that can be ordered online

>  I wanted to have someone help us with the Content, IA, and Accessibility for any CPAP content we end up making for this.

### Ticket Updates
- [x] **Added tags:** `sitewide CAIA` `site wide accessibility` `a11y-testing`
- [x] **Removed `Epic` tag for now**, since this is just a ticket at the moment. We can add it back and convert it to an Epic, should you need to tie tickets to this, or you can tie tickets to the overall Epic.
- [x] **Added sprint** in ZenHub and Projects
- [x] **Added dates**: open date, target date (end of sprint) and start date (feel free to change this one) in Projects
- [x] **Added priority** level in projects
- [x] **Moved from New intake** to "Ready" 
- [ ] **Move to** In Flight for Projects and In Process for ZenHub when starting
- [ ] **Artifacts needed to test/links**
- [ ] **Review Needed?** "There is a C/PAP supply order form[ (Form 2346b)](https://www.va.gov/vaforms/va/pdf/VA2346b%20(electronic).pdf)" - Intake Ticket Comment
- [ ] **Join Slack Channel:** va-cto-supply-reordering

cc: @eli-oat @coforma-terry

**Comment by eli-oat** [comment]
@JasmineYohannan I'll be tackling this review for you and your team -- when you have a chance can you follow up to let me know what needs to be reviewed, and provide links and staging logins I'll need to access that stuff to review?

Thanks!

**Comment by JasmineYohannan** [comment]
@eli-oat , apologies for the delay on this I was OOO last week. I am still coordinating with folks on the content team and will ping here/slack when I have more links/staging logins

**Comment by JasmineYohannan** [comment]
@eli-oat circling back here, apologies for the delay! screenshots for the  updated app below.  Nothing is in staging yet, but if you'd like to run locally or check out a test instance use [this branch/PR](https://github.com/department-of-veterans-affairs/vets-website/pull/25005). 

## Happy path views


### Intro page
![screencapture-localhost-3001-health-care-order-hearing-aid-batteries-and-accessories-order-form-2346-introduction-2023-07-26-14_40_05](https://github.com/department-of-veterans-affairs/va.gov-team/assets/7645362/294a156b-2dd6-48cf-b09e-cf40a911c15a)

### Veteran info page
![screencapture-localhost-3001-health-care-order-hearing-aid-batteries-and-accessories-order-form-2346-veteran-information-2023-07-26-14_40_27](https://github.com/department-of-veterans-affairs/va.gov-team/assets/7645362/cfb5aa56-894b-47c3-9a9e-247633e0ac44)

### Choose address
![screencapture-localhost-3001-health-care-order-hearing-aid-batteries-and-accessories-order-form-2346-address-2023-07-26-14_40_39](https://github.com/department-of-veterans-affairs/va.gov-team/assets/7645362/666c62a2-2bef-4a3d-847b-2531a2e75963)

### Supplies page w/ all supply types
![screencapture-localhost-3001-health-care-order-hearing-aid-batteries-and-accessories-order-form-2346-supplies-2023-07-26-14_42_17](https://github.com/department-of-veterans-affairs/va.gov-team/assets/7645362/c1d5f9da-b70c-4105-9f7b-90b93e5c4cb6)

### Review page
![screencapture-localhost-3001-health-care-order-hearing-aid-batteries-and-accessories-order-form-2346-review-and-submit-2023-07-26-14_44_52](https://github.com/department-of-veterans-affairs/va.gov-team/assets/7645362/5c63c9f7-cb11-4458-9e4a-76e582cf9d3b)

### Confirmation page
![screencapture-870b1b6fa27f4064abc4560ad9dfa45b-review-vetsgov-internal-health-care-order-hearing-aid-batteries-and-accessories-order-form-2346-confirmation-2023-08-17-15_28_55](https://github.com/department-of-veterans-affairs/va.gov-team/assets/7645362/4340075e-c70a-446d-8e18-e827640fa03e)





---

## Additional views


### Supplies page w/ no supplies
![screencapture-localhost-3001-health-care-order-hearing-aid-batteries-and-accessories-order-form-2346-supplies-2023-07-26-14_16_55 (1)](https://github.com/department-of-veterans-affairs/va.gov-team/assets/7645362/d55643ed-7e99-468f-aa5e-36220ed2e452)


### Supplies page w/ sleep apnea supplies only
![screencapture-localhost-3001-health-care-order-hearing-aid-batteries-and-accessories-order-form-2346-supplies-2023-07-26-14_43_10](https://github.com/department-of-veterans-affairs/va.gov-team/assets/7645362/851aea70-7cc2-4586-b06d-37e13ee890a1)

**Comment by eli-oat** [comment]
@JasmineYohannan is this available on staging? 

Can you provide any info I need to reach it there, including log in info if necessary? 

Thanks!

**Comment by sara-amanda** [comment]
## CAIA Note

### Static Entry Page
FYI @eli-oat looks like there was to be a staging review, relating to the CPAP (static entry page), today 10/3/2023 at 11:30 a.m.
- **Collaboration Cycle:** Sitewide CAIA, Medical Device Reordering Tool, new CPAP supplies static entry page
- However, staging review for the new CPAP supplies static entry page was canceled due to no findings. 
- The Drupal page was closely modeled on an existing static page that serves as an entry point to the same online form.
- [Collab Cycle GitHub ticket](https://www.google.com/url?q=https://github.com/department-of-veterans-affairs/va.gov-team/issues/66173%23issuecomment-1734284454&sa=D&source=calendar&ust=1696771372633961&usg=AOvVaw2Y6BotRzXoVYVXN60qc0zX)
cc: @coforma-terry

**Comment by coforma-terry** [comment]
Hi @JasmineYohannan - just following up on Eli's request above - do you have a link to your staging environment yet?
Thanks :)

**Comment by coforma-terry** [comment]
Thank you @JasmineYohannan 

CC: @eli-oat

**Comment by sara-amanda** [comment]
@strelichl just let us know when you hear. Thanks!

**Comment by coforma-terry** [comment]
Hi all - just following up on this @JasmineYohannan @strelichl 
Thank you!

**Comment by coforma-jamie** [comment]
I'm meeting with this team on May 14 to talk about auditing this form @coforma-terry @strelichl

**Comment by coforma-jamie** [comment]
## Meeting summary
Original goal of the ticket was to add CPAP to the supply reordering tool. The team put in a ticket to get CAIA’s support for content and accessibility. But CPAP ended up launching without a CAIA a11y looking at it.

Now, the team is at a crossroads on the project. They want to integrate CPAP into supplies and medications tools (preferred path), so that users can meet their needs from one place. But this may not happen for awhile. 

So the team is doing a "checkup" of sorts. They recently did usability testing on the current form, starting from the content/intro page. Didn’t find major issues; feedback was around step 1, to verify personal info (SSN, gender, name) - “why do i need to verify that if I’m logged in?” Can we integrate that directly? Also the “Finish this application later” link was missed by a few participants. [Full results here.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/supply-reordering-tool/research/2024-03-Supply-Reordering-Research/Findings%20and%20recommendations.md)

They're going to discuss their direction tomorrow - are we changing design in the next quarter, or not? Depending on the answer, they may do an audit, or an AT study, or neither.

## Action items

- [ ] @JasmineYohannan  will figure out if the tools has ever gone through a Staging Review / was evaluated for accessibility issues. If yes, and nothing’s changed since, CAIA won’t do an accessibility audit. If no, CAIA will do the audit.
- [ ] @coforma-jamie will send over some examples of AT studies done on forms (as a comment on this ticket).

**Comment by coforma-jamie** [comment]
Question for @JasmineYohannan - are you looking for examples of form AT studies that _anyone_ has done, or ones that _CAIA_ have been specifically involved in?

**Comment by coforma-jamie** [comment]
@JasmineYohannan, this might be helpful - [all the research tickets with the "Accessibility" label](https://github.com/department-of-veterans-affairs/va.gov-research-repository/projects/1?card_filter_query=label%3A%22accessibility%22). There are some form-related studies here that you can check out :)

**Comment by coforma-jamie** [comment]
## Next steps: Audit

I'll be auditing the Order hearing aid or CPAP supplies form, specifically the CPAP flow (the CPAP question on page 3), for accessibility issues. This flow didn't go through staging review, so this is serving as an accessibility "checkup."

- [Entry point](https://staging.va.gov/health-care/order-cpap-supplies/)
- Use ID.me, user+228@gmail.com (password in DM)

**Comment by coforma-jamie** [comment]
SK and I sent the completed audit spreadsheet to the team. They reviewed it and had no further questions, so we're closing this ticket.
